apiserver/admin_test.go

```diff
@@ -58,7 +60,8 @@ func (s *baseLoginSuite) SetUpTest(c *gc.C) {
 	s.JujuConnSuite.SetUpTest(c)
 	loggo.GetLogger("juju.apiserver").SetLogLevel(loggo.TRACE)
 
-	_, err := s.State.AddSpace("mgmt01", "", nil, false)
+	var err error
+	s.mgmtSpace, err = s.State.AddSpace("mgmt01", "", nil, false)
```

Comment:
Now that we filter by space IDs, spaces used for HA space or management space must exist in state for all such test scenarios.